app/sandpack/global-styles.js

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import { getSandpackCssText } from "@codesandbox/sandpack-react";
+import { useServerInsertedHTML } from "next/navigation";
+
+export const GlobalSandpackStyles = ({ children }) => {
+  useServerInsertedHTML(() => {
```

Comment:
😮 AMAZING!

Reply:
yes, this hook makes the job in `app` folder